src/docs/reference/modules/OpenId/README.md

```diff
@@ -49,9 +49,72 @@ Available settings are:
 - Enable Authorization Endpoint.
 - Enable Logout Endpoint.
 - Enable User Info Endpoint.
-- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3>
+- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3> 
+
+Typescript request example using axios:
+
+``` typescript 
+    
+  const data: any =
+  {
+        username: username,
+        password: password,
+        // must read from config
+        client_id: 'client_id',
+        client_secret: 'client_secret',
+        grant_type: 'password',
+        scope: 'profile email phone roles'
+  }
+
+  let formData = []
+  for (let it in data) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (data.hasOwnProperty(it)) {
+      formDatas.push(encodeURIComponent(it) + '=' + encodeURIComponent(data[it]))
+    }
+  } 
+  var res = axios.request({
+        url: ApiUrl + '/connect/token',
+        method: 'post',
+        data: formData.join('&'),
+        headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
+  })
+  return res
+```
+Complete example at <https://github.com/JiZhouSoft/Jizhousoft.OrchardCore.Sample/blob/cd1fd0c42893c6d0607f8642451d1a98649b9769/vue-ts-admin/src/api/users.ts#L10>
+
+
 - Allow Client Credentials Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.4>
-- Allow Authorization Code Flow: It requires that the Authorization and Token Endpoints are enabled. More info at <http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth>
+- Allow Authorization Code Flow: It requires that the Authorization and Token Endpoints are enabled. More info at <http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth> 
+
+Typescript request example using axios :
+
+``` Typescript     
+
+  const data: any =
+    {
+      "grant_type": "code",
```

Comment:
It's not a valid `grant_type`: the standard value is `authorization_code` for the authorization code flow.